libvcx/src/lib.rs

```diff
@@ -237,12 +237,26 @@ mod tests {
         set_institution();
         // GET REV REG DELTA BEFORE REVOCATION
         let (_, delta, timestamp) = ::utils::libindy::anoncreds::get_rev_reg_delta_json(&rev_reg_id.clone().unwrap(), None, None).unwrap();
-        println!("revoking credential");
+        info!("revoking credential");
         ::issuer_credential::revoke_credential(issuer_handle).unwrap();
         let (_, delta_after_revoke, _) = ::utils::libindy::anoncreds::get_rev_reg_delta_json(&rev_reg_id.unwrap(), Some(timestamp + 1), None).unwrap();
         assert_ne!(delta, delta_after_revoke);
     }
 
+    fn revoke_credential_local(issuer_handle: u32, rev_reg_id: Option<String>) {
```

Comment:
I think it'd be better if this function only takes `issuer_handle` as parameter and gets `rev_reg_id`  from credential itself, even if it means adding new getter on issuer credential